packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2262,7 +2264,15 @@ abstract class BuildContext {
   /// [InheritedWidget] subclasses that supports partial updates, like
   /// [InheritedModel]. It specifies what "aspect" of the inherited
   /// widget this context depends on.
-  T? dependOnInheritedWidgetOfExactType<T extends InheritedWidget>({ Object? aspect });
+  ///
+  /// When `stopAtLookUpBoundary` is set to true, null is returned if
+  /// no inherited widget of the specified type exists between this
+  /// [BuildContext] and the closest ancestor lookup boundary.
```

Comment:
reading this i would just ask myself "what is a lookup boundary". I think every mention of lookup boundary should link to `Element.isLookUpBoundary`.